packages/serverpod_shared/lib/src/password_manager.dart

```diff
@@ -71,4 +71,36 @@ class PasswordManager {
       return null;
     }
   }
+
+  /// Merge custom passwords with the existing password collection.
+  /// Custom passwords are loaded from the environment variables.
+  /// Throws an [ArgumentError] if the custom password configuration contains reserved keywords
```

Comment:
Question: What are the reserved keywords? Should we link to them here?

Reply:
Discussion(NIT): Maybe we could remove the `Custom` word? Isn't it enough with just `mergePasswords` or maybe `integratePasswords`?